docs/installation.rst

```diff
@@ -0,0 +1,217 @@
+
+.. code-block:: shell
+
+    $ pip install "runhouse[azure]"
+    # Login
+    $ az login
+    # Set the subscription to use
+    $ az account set -s <subscription_id>
+    $ sky check
+
+For more info: `SkyPilot | Azure <https://skypilot.readthedocs.io/en/latest/getting-started/installation.html#azure>`
+
+
+Other Providers
+---------------
+
+For a full list of providers and configuration details: `SkyPilot documentation <>`.
+Some additional providers supported by Runhouse via SkyPilot include:
+
+- Kubernetes
+- Lambda Cloud
+- & more
+
+
+Cluster Lifecycle Management
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To ensure that you have full control over the availability of your clusters, and the cost associated
+with maintaining them, you can may find the following SkyPilot commands helpful:
+
+- ``sky status`` - Displays a list of all your clusters by name.
+- ``sky stop <NAME>`` - Stops an instance. This will not fully tear down the cluster in case you need to restart it.
+- ``sky down <NAME>`` - Fully tears down the instance. Best for when you are done using a cluster entirely.
+- ``ssh <NAME>`` - Easily SSH into your cluster. Here you can further examine Runhouse's server with ``runhouse status``.
+
+Example output from ``sky status``:
+
+.. parsed-literal::
+    :class: code-output
+
+    Clusters
+    NAME                     LAUNCHED    RESOURCES                                        STATUS  AUTOSTOP    COMMAND
+    rh-a10                   4 days ago  1x AWS(g5.2xlarge, {'A10G': 1}, ports=['8080'])  UP      30m (down)  llama3_tgi_ec2.py
+    sky-3201-matthewkandler  4 days ago  1x AWS(g5.xlarge, {'A10G': 1})                   UP      -           sky launch --gpus A10G:1 ...
+
+    Managed jobs
+    No in-progress managed jobs. (See: sky jobs -h)
+
+    Services
+    No live services. (See: sky serve -h)
+
+
+Authenticate with Runhouse Den
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To enable sharing features and telemetry through Runhouse Den, you can log in to your
+Runhouse Den account. Start by creating one via our `signup page </signup>`.
+You'll have the option to authenticate with your Google or Github account.
+
+Once you've created an account, you'll be able to access your Runhouse token on your `account page <https://www.run.house/account>`.
+
+Login to Runhouse in your terminal with the following command:
+
+.. code-block:: shell
+    runhouse login
+
+You'll be prompted to enter your token and, after your initial login, you will see the following propmts:
+
+- ``Download your Runhouse config to your local .rh folder? [Y/n]:`` - Updates your local config file from your settings on the cloud
+- ``Upload your local .rh config to Runhouse? [y/N]:`` - Use updates to your local config to modify your saved account settings
+
+If you are running Runhouse strictly in code (like in a notebook), you can also login to your account
+with the Python API:
+
+.. code-block:: python
+    import runhouse as rh
+    rh.login()
+
+Misc.
```

Comment:
this section feels a little out of place since it's really just about runhouse secrets management, maybe we put this in the secrets docs and link out to that directly from the examples where relevant?

Reply:
Yeah that's a good point - it was reoccurring in the examples I was working on but definitely only makes sense for very specific use cases... probably best to include there, or even just keep in specific examples? Maybe something a little more generic than this in the secrets docs if not already covered

Reply:
In the secrets api docs there's a section starting with: "You can also include a list of secrets in a Runhouse env object", I think this captures that, although I didn't see a great way for us to link to that directly

Reply:
Yeah I think maybe a separate project to clarify and break that up a bit more... could live on the envs and packages page too